src/index.ts

```diff
@@ -59,28 +60,25 @@ const feeder = new Feeder(
   { output },
 );
 
-if (argv.parser) {
-  feeder.useParser(argv.parser);
-}
-
 (async () => {
```

Comment:
Typescript has top-level await. So you don't need this Immediately Invoked Function Expression (IIFE is (async ()=>{})() ) here